be/src/util/bitmap_value.h

```diff
@@ -1617,7 +1629,7 @@ class BitmapValue {
         case BitmapTypeCode::BITMAP32:
         case BitmapTypeCode::BITMAP64:
             _type = BITMAP;
-            _bitmap = detail::Roaring64Map::read(src);
+            _bitmap = std::move(detail::Roaring64Map::read(src));
```

Comment:
`detail::Roaring64Map::read(src)` is a temporary value.
"moving a temporary object prevents copy elision"